openshift-infra/roles/vpc/tasks/main.yml

```diff
@@ -0,0 +1,136 @@
+---
+
+# roles/vpc/tasks/main.yml
+
+
+# First task : creating the VPC.
+# We are using the variables set in the vars.yml file.
+# The module gives us back its result,
+# which contains information about our new VPC. 
+# We register it in the variable my_vpc.
+
+- name:               Create VPC
+  ec2_vpc_net:
+    name:             "{{ vpc_name }}"
```

Comment:
This variable along with others are not defined, should they be in aws_vars.yml? or somewhere else?